rct229/ruleset_functions/baseline_systems/test_is_baseline_system_8.py

```diff
@@ -0,0 +1,223 @@
+from rct229.schema.validate import schema_validate_rmr
+
+SYS_8_RMD = {
+    "id": "ASHRAE229 1",
+    "ruleset_model_instances": [
+        {
+            "id": "RMD 1",
+            "buildings": [
+                {
+                    "id": "Building 1",
+                    "building_open_schedule": "Required Building Schedule 1",
+                    "building_segments": [
+                        {
+                            "id": "Building Segment 1",
+                            "zones": [
+                                {
+                                    "id": "Thermal Zone 1",
+                                    "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                    "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                    "terminals": [
+                                        {
+                                            "id": "VAV Air Terminal 8",
+                                            "is_supply_ducted": True,
+                                            "type": "VARIABLE_AIR_VOLUME",
+                                            "served_by_heating_ventilation_air_conditioning_system": "System 8",
```

Comment:
We might need to manage this carefully. Could you change the merge target to the schema update branch? We will need to make sure this has been handled correct before merging to the develop branch.

Reply:
Of course! Fixed.